scripts/pocket-change.js

```diff
@@ -380,4 +454,67 @@ export default class PocketChange {
       token.actor.sheet.render(true);
     }
   }
+
+  
+  /**
+   * Converts the provided token to a item piles lootable sheet
+   *
+   * @param {object} options
+   * @param {Token5e} options.token - the token to convert
+   * @param {number} options.userOption - (optional) the type of convertion by default is 1
```

Comment:
I don't love this concept of userOption being an integer... but I don't have a strong suggestion for an alternative.